ipynb/filter_sequences_for_blooms.ipynb

```diff
@@ -0,0 +1,246 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Early in the American Gut Project, it was observed that some organisms bloomed likely as a result of increased shipping time and delay between when samples were collected and when they were put on ice (more detail can be found [here](http://americangut.org/?page_id=277)). The purpose of this notebook is to apply the filter developed in order to bioinformatically subtract these observed bloom sequences from fecal samples. It is important to apply this filter when combining data with the American Gut as to remove a potential study-effect bias as all fecal data in the American Gut has had this filter applied. The specific steps covered are:\n",
```

Comment:
This is a bit nit-picky.

But the term "apply the filter" is a little ambiguous at first, since the "filter" has not been identified.

Is it possible to reword it to something like
"The purpose of this notebook is to illustrate how these organisms are identified and filtered out in our bioinformatics software pipeline."